src/components/Patient/AccountSheet.tsx

```diff
@@ -109,7 +119,7 @@ export function AccountSheetButton({
                     <ExternalLink className="size-4" />
                     {t("more_details")}
                   </Button>
-                  {canWrite && (
+                  {canWrite && canUpdateAccount && (
```

Comment:
we can remove canWrite now since we have added the correct permissions 👍 Remove from props as well